lib/generators/spotlight/install_generator.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable ClassLength
```

Comment:
Perhaps I'm being dense, but how would I disable a particular rule for just generators?  I'm guessing we don't want to disable all rules for generators, but I'm unfamiliar with how we would do that with particular rules.

Also, I suppose we could break this out into separate generators.  I'm trying to think of where the logical divide between generators would be.  So do you think something like a dependencies generator (add dependencies and run their generators) and a generator that moves/copies/and injects into app-level files?